### PR TITLE
update to holochain 041ad6

### PR DESCRIPTION
**Pull request by zippy**
- [x] update test dna
- [x] bump version for publishing to npm

### CODE REVIEW

**Review by Connoropolous — APPROVED** [review]

File: install-holochain.sh [hunk 1]
```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
 
 # REV=beaab9949fbff121ffd138f36c4e61c5693b8fd6
```
Comment (Connoropolous) [inline install-holochain.sh]:
```suggestion
```

sorry I think I did this by accident, left this in

File: test/e2e/fixture/zomes/foo/Cargo.toml [hunk 2]
```diff
@@ -10,4 +10,4 @@ crate-type = [ "cdylib", "rlib" ]
 
 [dependencies]
 serde = "=1.0.104"
-hdk3 = { git = "https://github.com/holochain/holochain", rev = "a1174a5a3f66c47f486be312d38604e49424afaf", package = "hdk3" }
+hdk3 = { git = "https://github.com/holochain/holochain", rev = "3dd53e4ca03c8c0815b4412ea8ebd16f2b19cd47", package = "hdk3" }
```
Comment (Connoropolous) [inline test/e2e/fixture/zomes/foo/Cargo.toml]:
```suggestion
hdk3 = { git = "https://github.com/holochain/holochain", rev = "041ad6c2fd88c06eace450074621be2bace0eb05", package = "hdk3" }
```

@zippy you have a different REV in here, this is the one you used up above.
These things should stay in sycn

Comment (zippy) [reply test/e2e/fixture/zomes/foo/Cargo.toml]: whoops

**Review by Connoropolous — CHANGES_REQUESTED** [review]
It's all good, except for that mismatched REV

**Review by Connoropolous — APPROVED** [review]
as long as CI passes.
I don't know if this was a big or little REV change

**Review by maackle — APPROVED** [review]

**Comment by Connoropolous** [comment]
@maackle can you publish?

**Comment by zippy** [comment]
> @maackle can you publish?

Done

**Pull request merged**